### PR TITLE
Add support for camel-casing keys

### PR DESCRIPTION
**Pull request by SimenB**

### CODE REVIEW

**Comment by af** [comment]
I'm not really a fan of this change– one thing I like is if I have an env var `FOO_BAR`, I can grep for `FOO_BAR` and see every place that uses it, even though I'm not accessing the env var directly.

However, I'm on board for adding a hook so that this kind of functionality can be layered on top by the consuming code. Maybe we could add `options.transformOutput()` instead?

**Comment by SimenB** [comment]
Sure, I can do that instead! 😄 

The use case is to transform the names for use as options into other APIs. `FOO_BAR` is fine within an app, but not as options passed to a module (your option here is named `strict`, not `STRICT`). While my team really likes envalid, other teams still use e.g. `convict`, and we all use common modules taking options.

Providing a transform function is no problem, though

**Comment by SimenB** [comment]
@af Updated 😄

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "validation"
   ],
   "devDependencies": {
+    "camelcase-keys": "4.0.0",
```
Comment (af) [inline package.json]: Let's remove this dependency (I'm allergic to extra deps :) and just write a simple transformer test instead

**Comment by SimenB** [comment]
@af Updated

**Comment by af** [comment]
Thanks!

**Comment by af** [comment]
Just published v2.4.0 with this feature

**Pull request merged**